query/task_manager.go

```diff
@@ -266,6 +270,10 @@ func (t *TaskManager) waitForQuery(qid uint64, interrupt <-chan struct{}, closin
 
 		t.queryError(qid, err)
 	case <-timerCh:
+		if(t.LogQueryTimeouts){
```

Comment:
`gofmt` again. Go does not use parentheses for single expression `if` statements.